Library/Formula/byobu.rb

```diff
@@ -19,6 +20,9 @@ class Byobu < Formula
   depends_on "tmux"
   depends_on "newt" => "with-python"
 
+  depends_on "automake" => :build
```

Comment:
That's originally what I was going to do, but it looked like the stable build also needed proper autotools since I didn't see a pre-generated configure script in the release tarball I looked at. Maybe I was just seeing things.

I'll rework it tonight into the conditional.